src/lib.rs

```diff
@@ -23,7 +23,6 @@ mod connector;
 mod deposit_event;
 #[cfg(feature = "contract")]
 mod engine;
-#[cfg(feature = "contract")]
 mod fungible_token;
```

Comment:
I removed it because I need the FungibleToken struct in tests, but I didn't want to move the struct out of the module. All the logic is still hidden behind the feature flag because it's not used outside the contract, the tests only need the structure itself.

Reply:
A lot of structures I used in integrations tests from `contract` feature.
I just copied it. If something will change in tested function logic, anytime I will need to change that test.  I think it's redundant to open modules only for tests, not for executable parts.

Reply:
I think duplicating structures in tests makes the code more difficult to maintain because future changes always need to be done in two places. If tests use the same structures as production then we know the tests are always testing the right thing.

During the tech debt cycle after connectors release I think it would be good to refactor things so that duplicating structures in tests is not needed.

Reply:
For integration tests, if we change structure we must change that test anytime. :dancers: 
So I do not agree.